### ISSUE TITLE
Binary Bot Please Help??  Get trade results in profit/loss from account statement using bot

### ISSUE THREAD

**Issue by Jaskown** [binary-com/binary-bot#3494]
Hello,

How can I get the last two trades results in profit/loss from the account statement??

**Comment by alanvito1** [comment]
Look your account

**Comment by Jaskown** [comment]
> Look your account

Thanks @alanvito1 

Sorry I didn't specify, I need to read the profit/loss of the last trades of the account using a bot.

**Comment by alanvito1** [comment]
![Screenshot_2021-12-17-06-59-07-711_com.android.chrome.jpg](https://user-images.githubusercontent.com/46165592/146526517-96874792-a7a6-438d-bb47-a0275373992a.jpg)

**Comment by alanvito1** [comment]
Look for profit table

**Comment by Jaskown** [comment]
Thanks again @alanvito1 

As I said earlier, I know where I can get the profit table  Let me rephrase my  request, how can I use the trades data from the account statement in a bot?

**Comment by ChinthakaJP** [comment]
Subscribe my Channel for new profitable bots : 
https://www.youtube.com/channel/UCYFCodv64R4w5087LM4Jq4A
